applications/IgaApplication/python_scripts/python_solvers_wrapper_iga.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+
+
+def CreateSolver(model, custom_settings):
```

Comment:
this file is a copy from StructuralMechanics, you don't have most of the things ...

Also it is not up-to-date with the latest developments

Reply:
Yes, you are right. That was the wrong version